### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard:"Name","FileSize" and "LastModified" buttons are not accessible by keyboard.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#3955]
**EnvironmentDetails:**
Application Name:AzureNotebooks_Jyupter
Login Url: notebooks.azure.com
BrowserDetails:
Edge:Microsoft Edge 41.16299.611.0
Chrome:Version 68.0.3440.106 
Operating System:Windows10
 
 **Steps to reproduce:**
1.Login with the appropriate credentials to azure notebook account.    
2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library
3.Click on Run or right click and select Run to run the notebook.
5.Navigate to the "Upload" link using the "Tab" key.
**Actual:
Issue1:**
Name","FileSize" and "LastModified" buttons are not accessible by keyboard.
**Issue2:**
Keyboard focus is not logical after the "Select..." comobo-box button.
**Expected:
Issue1:**
"Name","FileSize" and "LastModified" buttons are not accessible by keyboard.
**Issue2:**
Keyboard focus should move to the "Name","FileSize" and "LastModified" buttons
**UserImpact:**
User will face difficulty while traversing through the page. 
**Attachment:**
[A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard12.pptx](https://github.com/jupyter/notebook/files/2399898/A11y_AzureNotebooks_Jupyter_FilesTab_Keyboard12.pptx)

**Comment by ednut15** [comment]
Fixed by pull request #4703  (Solved issue #3957 )

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
